examples/UKCore-Condition-Extension-CodingSCTDescId-SN-Amoxicillin-Ex.xml

```diff
@@ -0,0 +1,17 @@
+<Condition xmlns="http://hl7.org/fhir">
+	<id value="UKCore-Condition-Extension-CodingSCTDescId-SN-Amoxicillin-Ex"/>
+	<!-- **************Snippet start************** -->
+	<code>
+		<coding>
+			<system value="https://dmd.nhs.uk/"/>
+			<code value="323509004"/>
+			<display value="Amoxicillin 250mg capsules"/>
```

Comment:
Example makes no real sense, you wouldn't send a medication concept in a condition